source/includes/appschema-configuration.rst

```diff
@@ -135,7 +135,7 @@ Configuration
    * - | ``custom_user_data_config``
        | Document
      - A document that contains configuration options for
-       :doc:`custom user data </users/enable-custom-user-data>`.
+       :doc:`custom user data </users/custom-metadata>`.
```

Comment:
for all these link changes, could you replace the `:doc:` directives with `:ref:` directives?

it's more maintainable to use refs, and generally the practice we follow these days. the doc directives are probably only in the site as a legacy, and should have been changed a while ago.